src/components/dom.js

```diff
@@ -112,6 +112,24 @@ define([], function () {
         return windowSize;
     }
 
+    var standardWidths = [480, 720, 1280, 1440, 1920, 2560, 3840, 5120, 7680];
+    function getScreenWidth() {
+        var width = window.innerWidth;
+        var height = window.innerHeight;
+
+        if (height > width) {
+            /* If we're in portrait, compute the proper width for this height,
+               taking 16:9 as the most common ratio */
+            width = height * 1.78;
```

Comment:
```suggestion
            width = height * 16.0 / 9.0;
```
for clarity?

Reply:
I think 1.78 is fairly standard.

What does @jellyfin/web think ?

Reply:
I wouldn't have known, so this makes more sense to me. You could probably remove the comment with this change.